services/src/main/java/org/keycloak/protocol/oidc/OIDCConfigAttributes.java

```diff
@@ -21,6 +21,8 @@
     public static final String USER_INFO_RESPONSE_SIGNATURE_ALG = "user.info.response.signature.alg";
 
     public static final String REQUEST_OBJECT_SIGNATURE_ALG = "request.object.signature.alg";
+    public static final String REQUEST_OBJECT_ENCRYPTION_ALG = "request.object.encryption.alg";
+    public static final String REQUEST_OBJECT_ENCRYPTION_ENC_ALG = "request.object.encryption.enc.alg";
```

Comment:
According to https://openid.net/specs/openid-connect-registration-1_0.html#ClientMetadata, it might be better to be 
```
public static final String REQUEST_OBJECT_ENCRYPTION_ENC = "request.object.encryption.enc";
```
WDYT?